<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>720</width>
    <height>420</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>720</width>
    <height>420</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>720</width>
    <height>420</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout" stretch="1,5">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QWidget" name="widget_4" native="true">
     <layout class="QHBoxLayout" name="_2">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QLabel" name="currentTask">
        <property name="font">
         <font>
          <family>仿宋</family>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(255, 220, 192);</string>
        </property>
        <property name="text">
         <string>当前目标：</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignmentFlag::AlignCenter</set>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <property name="spacing">
      <number>0</number>
     </property>
     <item>
      <widget class="QWidget" name="widget" native="true">
       <layout class="QVBoxLayout" name="verticalLayout_2" stretch="1,4">
        <property name="spacing">
         <number>0</number>
        </property>
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <item>
         <widget class="QLabel" name="label">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="font">
           <font>
            <family>仿宋</family>
            <pointsize>12</pointsize>
           </font>
          </property>
          <property name="text">
           <string>今日任务</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignmentFlag::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QWidget" name="todayTasks" native="true">
          <widget class="QPushButton" name="todayButton">
           <property name="geometry">
            <rect>
             <x>180</x>
             <y>220</y>
             <width>50</width>
             <height>50</height>
            </rect>
           </property>
           <property name="minimumSize">
            <size>
             <width>50</width>
             <height>50</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>50</width>
             <height>50</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">QPushButton {
    background-color: #4CAF50; /* 背景颜色 */
    color: white;              /* 字体颜色 */
    border: none;              /* 无边框 */
    border-radius: 25px;      /* 圆角半径 */
    font-size: 24px;          /* 字体大小 */
}
QPushButton:hover {
    background-color: #45a049; /* 悬停时的背景颜色 */
}
QPushButton:pressed {
    background-color: #388e3c; /* 按下时的背景颜色 */
}</string>
           </property>
           <property name="text">
            <string>+</string>
           </property>
          </widget>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QWidget" name="widget_2" native="true">
       <layout class="QVBoxLayout" name="verticalLayout_3" stretch="1,4">
        <property name="spacing">
         <number>0</number>
        </property>
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <item>
         <widget class="QLabel" name="label_2">
          <property name="font">
           <font>
            <family>仿宋</family>
            <pointsize>12</pointsize>
           </font>
          </property>
          <property name="text">
           <string>本周任务</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignmentFlag::AlignCenter</set>
          </property>
          <property name="openExternalLinks">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QWidget" name="weekTasks" native="true">
          <widget class="QPushButton" name="weekButton">
           <property name="geometry">
            <rect>
             <x>180</x>
             <y>220</y>
             <width>50</width>
             <height>50</height>
            </rect>
           </property>
           <property name="minimumSize">
            <size>
             <width>50</width>
             <height>50</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>50</width>
             <height>50</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">QPushButton {
    background-color: #4CAF50; /* 背景颜色 */
    color: white;              /* 字体颜色 */
    border: none;              /* 无边框 */
    border-radius: 25px;      /* 圆角半径 */
    font-size: 24px;          /* 字体大小 */
}
QPushButton:hover {
    background-color: #45a049; /* 悬停时的背景颜色 */
}
QPushButton:pressed {
    background-color: #388e3c; /* 按下时的背景颜色 */
}</string>
           </property>
           <property name="text">
            <string>+</string>
           </property>
          </widget>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QWidget" name="widget_3" native="true">
       <layout class="QVBoxLayout" name="verticalLayout_4" stretch="1,4">
        <property name="spacing">
         <number>0</number>
        </property>
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <item>
         <widget class="QLabel" name="label_3">
          <property name="font">
           <font>
            <family>仿宋</family>
            <pointsize>12</pointsize>
           </font>
          </property>
          <property name="text">
           <string>本月任务</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignmentFlag::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QWidget" name="monthTasks" native="true">
          <widget class="QPushButton" name="monthButton">
           <property name="geometry">
            <rect>
             <x>180</x>
             <y>220</y>
             <width>50</width>
             <height>50</height>
            </rect>
           </property>
           <property name="minimumSize">
            <size>
             <width>50</width>
             <height>50</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>50</width>
             <height>50</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">QPushButton {
    background-color: #4CAF50; /* 背景颜色 */
    color: white;              /* 字体颜色 */
    border: none;              /* 无边框 */
    border-radius: 25px;      /* 圆角半径 */
    font-size: 24px;          /* 字体大小 */
}
QPushButton:hover {
    background-color: #45a049; /* 悬停时的背景颜色 */
}
QPushButton:pressed {
    background-color: #388e3c; /* 按下时的背景颜色 */
}</string>
           </property>
           <property name="text">
            <string>+</string>
           </property>
          </widget>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
